compiler/rustc_feature/src/removed.rs

```diff
@@ -134,6 +134,8 @@ declare_features! (
            which is available from cargo build scripts with `cargo:rustc-link-arg` now")),
     /// Allows using `#[main]` to replace the entrypoint `#[lang = "start"]` calls.
     (removed, main, "1.53.0", Some(29634), None, None),
+    (removed, pub_macro_rules, "1.53.0", Some(78855), None,
```

Comment:
I guess since is the first version in which the feature would be removed (which would be 1.53) but I wasn't 100% sure, so please check this in particular when reviewing.